variables.tf

```
@@ -1,6 +1,7 @@
 variable "project" {
   description = "Name of the PE architecture project"
   type        = string
+  default     = "ape"
```

It seems we don't have a default project name for the [terraform-google-pe_arch](https://github.com/puppetlabs/terraform-google-pe_arch/blob/main/variables.tf), but should we have a self-explanatory name? 

@ody what do you think about this?

```suggestion
  default     = "puppet_enterprise_infrastructure"
```

---

Has to be short, else the constructed names are to long and AWS API complains, failing the Terraform run.

>   Error: "name" cannot be longer than 32 characters: "pe-compiler-elb-slice-cody-5f8a3b"
> 
>     on modules/loadbalancer/main.tf line 7, in resource "aws_elb" "pe_compiler_elb":
>      7:   name            = "pe-compiler-elb-${var.project}-${var.id}"

We should probably move `project` on the AWS module to a tag because the generated `id` already makes the resource names unique. Probably the primary reason it exists is for consistency of API because on GCP it is a required parameter.

---

If we can merge this ahead of the sprint end, I'll ticket the task of improving the uses of project within this aws module

---

Got it, and I agree the `ID` may be more useful on the resource name for searching purposes.

Sold, let's improve this in a separate PR. Thanks @ody :)